vagrant/dev/bootstrap.sh

```diff
@@ -25,3 +25,8 @@ yum install -y \
 	libffi-devel openssl-devel \
 	yum-utils \
 	pypy pypy-devel
+
+
+# dpkg isn't currently available upstream
```

Comment:
It isn't *quite* available. It is currently a proposed update, and should land in a week or two.